src/deluge/dsp/util.hpp

```diff
@@ -87,4 +89,22 @@ void foldBuffer(q31_t* startSample, q31_t* endSample, q31_t foldLevel) {
 		currentSample += 1;
 	} while (currentSample < endSample);
 }
+
+/* Original code for fastlog2f by Dr. Paul Beckmann from the ARM community
+forum, adapted from the CMSIS-DSP library About 25% performance increase over
+std::log10f
+*/
+inline float log2fast(float f) {
```

Comment:
have you benchmarked this? I would be surprised if it's actually faster. The arm built in appears to use the same number of polynomial approximation rounds, but uses direct conversion of the ieee754 float value to exponent/mantissa as it's initial estimate instead of frexp

Reply:
it might not be, this was originally written for Cortex-M I think. I can do a Cortex-A benchmark with a stub on a beaglebone or something if you really want

Reply:
I'd prefer to just use std::log. I'm open to keeping this if it benchmarks significantly better but comparing source code and assembly I believe this only saves two multiplies and adds a function call compared to std::log. That would be a large speed up without hardware floats but might even be a loss for us.

Reply:
yeah that's fair, I'll take it out.